Running in wrapper mode for '96'

Running python job.

[name: "CPU-QUOTA"
int_val: 45
int_val: 42
int_val: 41
int_val: 60
int_val: 71
int_val: 24
int_val: 84
, name: "DISK"
int_val: 18
int_val: 10
int_val: 14
int_val: 31
int_val: 40
int_val: 36
int_val: 35
, name: "MEMORY"
int_val: 38
int_val: 10
int_val: 9
int_val: 34
int_val: 6
int_val: 28
int_val: 28
, name: "NET"
int_val: 5
int_val: 31
int_val: 13
int_val: 14
int_val: 31
int_val: 13
int_val: 17
]
The paramters are {u'CPU-QUOTA': array([45, 42, 41, 60, 71, 24, 84]), u'NET': array([ 5, 31, 13, 14, 31, 13, 17]), u'DISK': array([18, 10, 14, 31, 40, 36, 35]), u'MEMORY': array([38, 10,  9, 34,  6, 28, 28])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b762320>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.45 seconds.
